obspy/io/sac/util.py

```diff
@@ -159,19 +159,24 @@ def is_same_byteorder(bo1, bo2):
     return (bo1.lower() in le) == (bo2.lower() in le)
 
 
-def _clean_str(value):
+# This fn is used in two places: in SACTrace.read, to sanitize strings for
+# SACTrace, and in sac_to_obspy_header, to sanitize strings for Trace that the
+# user may have manually added.
```

Comment:
:+1: for making it a docstring. Also, I only see this function used in `SACTrace.read`, actually.